packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -43,11 +58,18 @@ export class Checkbox extends BaseComponent<ICheckboxProps, ICheckboxState> impl
 
     this._id = getId('checkbox-');
     this.state = {
-      isFocused: false,
       isChecked: !!(props.checked !== undefined ? props.checked : props.defaultChecked)
     };
   }
 
+  public componentWillReceiveProps(newProps: ICheckboxProps) {
```

Comment:
Yes, this is not related to styling. This is not also a bug fix per se. Because, we did away with the input of checkbox type, and moved to using a button, we have to take care of the state even when the case of a controlled checkbox component. Previously, we did not care about having the right state in case of a controlled component.